crates/astria-sequencer-relayer/src/relayer/celestia_client/celestia_keys.rs

```diff
@@ -124,8 +124,10 @@ mod tests {
     #[test]
     fn should_fail_to_construct_from_missing_file() {
         let error = CelestiaKeys::from_path("missing").unwrap_err();
-        // allow: `assert!(matches!(..))` provides poor feedback on failure.
-        #[allow(clippy::manual_assert)]
+        #[expect(
+            clippy::manual_assert,
+            reason = "`assert!(matches!(..))` provides poor feedback on failure"
+        )]
```

Comment:
Can you make a followup issue for these? It'd be nice to have some kind of `assert_error_variant` utility function in these tests.

Reply:
https://github.com/astriaorg/astria/issues/1581